pkg/porter/publish.go

```diff
@@ -328,3 +343,14 @@ func (p *Porter) rewriteImageWithDigest(InvocationImage string, digest string) (
 	}
 	return fmt.Sprintf("%s@%s", named.Name(), digest), nil
 }
+
+// refreshCachedBundle will store a bundle anew, if a bundle with the same tag is found in the cache
+func (p *Porter) refreshCachedBundle(bun *bundle.Bundle, tag string, rm relocation.ImageRelocationMap) error {
+	if _, _, found, _ := p.Cache.FindBundle(tag); found {
+		_, _, err := p.Cache.StoreBundle(tag, bun, rm)
+		if err != nil {
```

Comment:
i think this shouldn't be a fatal error? how about a warning?

Reply:
Good call; agreed!  Will update.